tron/utils/exitcode.py

```diff
@@ -0,0 +1,26 @@
+# TRON-1826 Refactored code and moved exit codes to a new file to be imported in both Kubernetes.py and actionrun.py
```

Comment:
suggestion: it's probably fine to obviate this comment - future CI ideally won't know/need to know where this came from originally :)